### DISCUSSION TITLE
Search bar for User Repositories

### DISCUSSION THREAD

**Discussion by justinmreina** [community/community#4364] [Mobile] [answered]
Hi,
Can we please have a search bar for personal repositories in the app? Similar to how the web interface displays "Find a Repository...", above the personal repositories 

Feat Request:
• "Find a Repository..." search box at the top of repository listing under 'Profile'->'Repositories' for the mobile iOS app

Would be great! 

Thanks,
Justin Reina

**Comment by taki-on** [comment, marked as answer]
Hi @justinmreina. I will forward the idea to the product team, so that we can discuss it internally. Thank you very much for your feedbacks and ideas. Please keep them coming to us!

**Comment by devpolo** [comment]
Good idea ! 👍
